oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -51,52 +51,96 @@ Reports come into security@openedx.org
 The email will be added to SECURITY.md in Github, and an email link will be provided on openedx.org along with the Open edX
 security policy.
 
-Disclosure triage by on-call Security Working Group Member
-----------------------------------------------------------
+Reports are triaged by the on-call security working group member
+----------------------------------------------------------------
 The on-call security working group member will triage the incoming report.
 
-Triage responsibilities include:
+They will:
 
-* On-call person stays with security issue until it is resolved
-* On-call responsibilities
+* Verify the report
+* Reply to the reporter
+* Forward operator-specific vulnerabilities to any relevant Open edX operators
+* Score the severity of the disclosure
+* Identify the affected repositories
+* Open a GitHub repository security advisory for the vulnerability in the relevant repositories
+* Notify the Open edX maintainers of those repositories of the vulnerability
 
-  * Triage and calculate severity of issue
-  * Notify and work with SME for the impacted area(s)
-  * Get ticket prioritized and resolved (i.e. project manage the ticket(s) in SME backlogs)
+The security working group gives security severity and remediation advice to maintainers
```

Comment:
```suggestion
Issue Triage and Routing
```
These section titles are very verbose, here's one suggestion for a shorter one but if you have a different idea, feel free to use that also.

Reply:
Great suggestion.